projects/cashmere/src/lib/sass/form-field.scss

```diff
@@ -138,6 +138,10 @@ $wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
     vertical-align: baseline;
 }
 
+@mixin hc-form-field-no-label() {
+    border-top: none !important;
```

Comment:
I can't see a reason why anybody would want to override here and put their own border on this particular DOM element, but I still  have to ask - any chance we can rearrange things or make selectors more specific in some way to avoid the use of !important?

Reply:
Cool - looks like I don't need `important` to override the base style (thought I did).  Removing it.